src/twisted/internet/defer.py

```diff
@@ -1512,6 +1589,12 @@ def thingummy():
         def loadData(url):
             response = yield makeRequest(url)
             return json.loads(response)
+    You can cancel the L{Deferred} returned from your L{inlineCallbacks}
+    generator before it is fired by your generator completing (either by
+    reaching its end, a C{return} statement, or by calling L{returnValue}).
+    This will cause any L{Deferred} being awaited upon in a C{yield} statement
+    to be cancelled, with whatever effects would result from that (often a
+    C{CancelledError} being raised from the C{yield}, but not always).
```

Comment:
This is vague.  A `CancelledError` will be raised from the `yield`ed `Deferred` that has been canceled if that `Deferred` does not otherwise suppress it.